dashboard/app/views/levels/editors/_curriculum_reference.haml

```diff
@@ -11,6 +11,6 @@
 .field
   = f.label :reference, 'Curriculum Reference'
   %p
-    String representing the URL on docs.code.org or curriculum.code.org that you want to embed,
+    String representing the URL on studio.code.org/docs or studio.code.org/curriculum that you want to embed,
```

Comment:
its a very specific link that I think also has to do with the student facing lesson plan. here is an example studio.code.org/curriculum/cspilot/unit6/4/overview/index.html

Reply:
studio.code.org/docs also gives a 404 as does curriculum.code.org/documentation - we probably need to build a landing page for documentation?

Reply:
ah, interesting... I picked the two urls I listed because curriculum.code.org and curriculum.code.org/hoc/ are both valid URLs. but I get what you're saying now.  nice find!